boilerplate/backend/python-fastapi/requirements.txt

```diff
@@ -1,3 +1 @@
-fastapi==0.63.0
-uvicorn==0.16.0
-supertokens-python
+supertokens-python[fastapi]
```

Comment:
nope. I prefer if we just do supertokens-python here, and hten have fastapi and uvicorn as a separate dependency. Same goes for others.